android/vpnClient/src/main/java/org/mozilla/firefox/vpn/qt/PackageManagerHelper.java

```diff
@@ -39,7 +39,9 @@ private static String getAllAppNames(Context ctx) {
     for (int i = 0; i < packs.size(); i++) {
       PackageInfo p = packs.get(i);
       // Do not add ourselves and System Apps to the list, unless it might be a browser
-      if ((!isSystemPackage(p,pm) || browsers.contains(p.packageName))
+      // or allowlisted
+      if ((!isSystemPackage(p, pm) || browsers.contains(p.packageName)
+              || isAllowListed(p.packageName))
```

Comment:
Now that we have jUnit test infrastructre, can we add tests for this?

Reply:
>:c probably

Reply:
![Untitled](https://github.com/mozilla-mobile/mozilla-vpn-client/assets/9611612/36a88bcf-3c6d-4ba3-b3b4-6ff8a025f672)